tests/search/search_aggregates_v8/test_aggregation/search_clients_daily_v8_bq_test.schema.json

```diff
@@ -2,102 +2,96 @@
   {
     "type": "DATE",
     "name": "submission_date",
-    "mode": "REQUIRED"
+    "mode": "NULLABLE"
   },
   {
-    "type": "INTEGER",
-    "name": "profile_creation_date",
-    "mode": "REQUIRED"
+    "type": "STRING",
+    "name": "client_id",
+    "mode": "NULLABLE"
   },
-
   {
     "type": "STRING",
-    "name": "app_version",
+    "name": "engine",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "country",
+    "name": "source",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "distribution_id",
+    "name": "country",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "engine",
+    "name": "addon_version",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "locale",
+    "name": "app_version",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "search_cohort",
+    "name": "distribution_id",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "source",
+    "name": "locale",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "default_search_engine",
+    "name": "user_pref_browser_search_region",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "default_search_engine_data_submission_url",
+    "name": "search_cohort",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "user_pref_browser_search_region",
+    "name": "os",
     "mode": "NULLABLE"
   },
   {
-    "type": "INTEGER",
-    "name": "max_concurrent_tab_count_max",
+    "type": "STRING",
+    "name": "os_version",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "default_search_engine_data_load_path",
+    "name": "channel",
     "mode": "NULLABLE"
   },
   {
     "type": "INTEGER",
-    "name": "tab_open_event_count_sum",
+    "name": "profile_creation_date",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "client_id",
+    "name": "default_search_engine",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "os",
+    "name": "default_search_engine_data_load_path",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "os_version",
+    "name": "default_search_engine_data_submission_url",
     "mode": "NULLABLE"
   },
   {
     "type": "STRING",
-    "name": "channel",
-    "mode": "NULLABLE"
-  },
-  {
-    "type": "FLOAT",
-    "name": "active_addons_count_mean",
+    "name": "sample_id",
```

Comment:
`sample_id` should be an integer here

Reply:
`sample_id` is a string in the `search_clients_daily_v7` even though the values are integers.  I'm not sure it's worth changing.

Reply:
Unless we decided that all `sample_id` will be integers from now on

Reply:
we did decide that, also they are integers in main_summary (in bigquery), so they will be integers in search_clients_daily_v8, so they will be integers here